apps/frontend/src/pages/root/components/user-wheel/libs/components/edit-mode-switch/edit-mode-switch.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from "~/libs/components/components.js";
+
+import { type EditMode } from "../../types/types.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	currentMode: EditMode;
+	handleModeToggle: () => void;
+};
+
+const EditModeSwitch: React.FC<Properties> = ({
+	currentMode,
+	handleModeToggle,
+}: Properties) => {
+	return (
+		<div className={styles["container"]}>
+			<Button
+				isDisabled={currentMode === "manual"}
+				isSelected={currentMode === "manual"}
```

Comment:
Move to a separate variable

Reply:
Not fixed

Reply:
Those are on local, I re-requested the review instead of writing to the chat asking to reply to the comment about scores entity